modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -40,31 +40,31 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 nameserver=4.4.4.41
 ----
 
-a|Specify multiple network interfaces by specifying multiple `ip=` entries.
-
+|Specify multiple network interfaces by specifying multiple `ip=` entries.
```

Comment:
Thanks, @codyhoag. This change confuses me because it rewrites what was there before and I don't know what ifdefs to apply when I add the missing content back.

Reply:
Upon further review, I don't actually think ifdefs are necessary. Just a restore to the original content in this file.